src/vehicle_move.cpp

```diff
@@ -569,6 +616,15 @@ void vehicle::thrust( int thd, int z )
     }
 }
 
+/**
+ * If the amount is positive, increase the cruise velocity by the amount, but not above the maximum
+ * velocity. If the amount is negative, decrease the cruise velocity by the amount, but not below the
+ * maximum reverse velocity
+ * 
+ * @param amount The amount to change the cruise velocity by.
+ * 
+ * @return the cruise velocity.
```

Comment:
The function does not return anything. It's declared `void`. Same applies to various functions below.

Reply:
Ok. Thx!